docs/usage.md

```diff
@@ -922,92 +926,131 @@ Specify the minimum percent identity (or similarity) a squence must have to the
 #### `--malt_mode`
 
 Use this to run the program in 'BlastN', 'BlastP', 'BlastX' modes to align DNA and DNA, protein and protein, or DNA reads against protein references respectively.
-respectively. Ensure your database matches the mode. Check the [MALT manual](http://ab.inf.uni-tuebingen.de/data/software/malt/download/manual.pdf) for more details. Default: 'BlastN'
+respectively. Ensure your database matches the mode. Check the [MALT manual](http://ab.inf.uni-tuebingen.de/data/software/malt/download/manual.pdf) for more details. Default: 'BlastN'  
+
+Only when `--metagenomic_tool malt`
```

Comment:
```suggestion
Only when `--metagenomic_tool malt` is also supplied
```
While I thought this would've been implicit with the paramtere name, better to be explicit ;). The complete sentence can be applied to all below as well.

Reply:
Changed every occurence to your suggestion